.ci/setup_env_centos.sh

```diff
@@ -105,6 +105,16 @@ main()
 	# GNU parallel not available in Centos repos, so build it instead.
 	build_install_parallel
 
+	# Extra CRI-O dependencies
+	if [ "$centos_version" >= "8" ]; then
+		curl -OkL http://mirror.centos.org/centos/7/os/x86_64/Packages/btrfs-progs-4.9.1-1.el7.x86_64.rpm
+		sudo -E rpm -ivh btrfs-progs-4.9.1-1.el7.x86_64.rpm
```

Comment:
oii, why don't use the btrfs for centos 8? https://centos.pkgs.org/8/elrepo-testing-x86_64/

Reply:
@devimc centos 8 at http://mirror.centos.org/centos/8/ does not have anymore the btrfs

Reply:
@devimc the `elrepo-testing` is a third party repo

Reply:
@GabyCT what other filesystem could we use instead of btrfs ?

Reply:
@jcvenegas `btrfs` is a dependency to build CRI-O, I am not sure if there is another filesystem that we could use

Reply:
@GabyCT, we can simply skip btrfs without issues, just pass `exclude_graphdriver_btrfs` to the BUILDTAGS.